Tests/Serializer/CallbackTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use \FOS\ElasticaBundle\Serializer\Callback;
+use \Symfony\Component\Serializer\Serializer;
+
+class CallbackTest extends PHPUnit_Framework_TestCase
+{
+    public function testSerializerMustHaveSerializeMethod()
+    {
+        $callback = new Callback();
+        $this->setExpectedException(RuntimeException::class, 'The serializer must have a "serialize" method.');
+        $callback->setSerializer(new \stdClass());
+    }
+
+    public function testSetGroupsWorksWithValidSerializer()
+    {
+        $callback = new Callback();
+        $serializer = $this->prophesize(Serializer::class);
```

Comment:
Hi @mickadoo, I've noticed a failure of tests ``` $ PHP Fatal error:  Call to undefined method CallbackTest::prophesize() in /home/travis/build/FriendsOfSymfony/FOSElasticaBundle/Tests/Serializer/CallbackTest.php on line 18``` this is due to phpunit version because phrophecy is included into phpunit from v4.5. I hope to help you with your PR, best regards!